charger/easee.go

```diff
@@ -36,12 +41,15 @@ type Easee struct {
 	*request.Helper
 	charger       string
 	site, circuit int
-	status        easee.ChargerStatus
-	updated       time.Time
```

Comment:
Ohne Timestamp  kannst du nie wissne, ob überhaupt noch Daten kommen. Ich würde zumindest einen timeout implementieren.

Reply:
Hab den Timestamp wieder eingebaut und aktualisiere ihn auch. Wie stellst du dir denn den Timeout vor? Eine Fehlermeldung die in einem Getter fliegt? Wäre mir gar nicht sicher auf welchen Wert man das bei einer Streaming API stellen sollte. Die schicken ja nur Daten wenn sich wirklich was verändert. So etwas wie einen Heartbeat hab ich bislang noch nicht gesehen.

Reply:
Ich würds einfach machen. Heartbeat ist wenn Daten kommen.

Reply:
Ok, das ist nun eingebaut. Allerdings wird das erstmal nur erfasst und der Timestamp nicht weiter ausgewertet.